content/en/blog/_posts/2022-4-26-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes 1.24: <release name>"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md): Mickey Boxell (Oracle), Kat Cosgrove (Pulumi), Debabrata Panigrahi, Parthvi Vala (Red Hat)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, and 13 enhancements are entering alpha. Also, 2 features have been deprecated, and the [dockershim has been removed](https://kubernetes.io/blog/2022/02/17/dockershim-faq/).     
+
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+After an initial deprecation in v1.20, the dockershim has been removed from the kubelet. From v1.24 and up, if you are currently relying on Docker Engine as your container runtime, you will need to either use one of the other supported runtimes (such as containerd or CRI-O) or use cri-dockerd. For more information about ensuring your cluster is ready for this removal, please see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
+
+### gRPC Probes Graduates to Beta
+With Kubernetes 1.24, the gRPC probes functionality has entered beta and is available by default.
+
+You can now configure startup, liveness, and readiness probes for your gRPC app without exposing an HTTP endpoint or using an extra executable, natively within Kubernetes.
+
+### Kubelet Credential Provider graduates to Beta
+Released as Alpha in Kubernetes v1.20, the Kubelet Credential Provider has now graduated to Beta. This allows the kubelet to dynamically retrieve credentials for a container image registry using exec plugins, communicating through stdio using Kubernetes versioned APIs, rather than storing them statically on disk.
```

Comment:
We could link to https://kubernetes.io/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/ here.

Reply:
Would it be better to link to that or to the issue? https://github.com/kubernetes/enhancements/issues/2133